yt_dlp/extractor/rutube.py

```diff
@@ -82,11 +88,18 @@ def _extract_formats(self, options, video_id):
                     'format_id': format_id,
                 })
         for hls_url in traverse_obj(options, ('live_streams', 'hls', ..., 'url', {url_or_none})):
-            formats.extend(self._extract_m3u8_formats(hls_url, video_id, ext='mp4', fatal=False))
-        return formats
-
-    def _download_and_extract_formats(self, video_id, query=None):
-        return self._extract_formats(
+            fmts, subs = self._extract_m3u8_formats_and_subtitles(hls_url, video_id, ext='mp4', fatal=False)
```

Comment:
I know this is basically old code, but would be good to add an `m3u8_id`
```suggestion
            fmts, subs = self._extract_m3u8_formats_and_subtitles(
                hls_url, video_id, 'mp4', fatal=False, m3u8_id='hls')
```